test/studies/arkouda/sub_test

```diff
@@ -38,6 +41,17 @@ else
   export "PATH=${ARKOUDA_HOME}/dep/hdf5-install/bin:$PATH"
 fi
 
+# CHPL_TEST_ARKOUDA_DISABLE_MODULES is a colon separated list of modules to
+# disable.  Disable these modules by commenting them out in ServerModules.cfg
+if [ ! -z "${CHPL_TEST_ARKOUDA_DISABLE_MODULES+x}" ] ; then
+  IFS=":"
+  for mod in $CHPL_TEST_ARKOUDA_DISABLE_MODULES; do
+    cmd="s/^\s*$mod/#$mod/g"
+    sed -i $cmd ServerModules.cfg
```

Comment:
Note that `sed -i` is not portable. It should work on the XC config you want to test on, but it won't work on say mac so we should make sure this this is only called when `CHPL_TEST_ARKOUDA_DISABLE_MODULES` is actually set.

Reply:
Looking at the man page for it on my mac, it looks like you can use `-i` but for whatever reason their version requires you pass an argument for the extension you want for the backup file (but that argument can be a blank string if you don't actually want a backup). You can pass this argument on the GNU version too but it's optional.

So, it looks like I can change to `sed -i'' -e $cmd ServerModules.cfg` and it should work on the XC I care about and Mac alike.